src/styles/index.scss

```diff
@@ -1,3 +1,130 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.card {
+  border: 1px solid #f3f3f3;
+  box-sizing: border-box;
+  width: 200px;
+
+  padding: 32px 16px 16px;
+
+  &__icon {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__info {
+    text-align: left;
+    color: rgb(6, 11, 53);
+
+    line-height: 18px;
+    font-size: 12px;
+    font-weight: 500;
+    margin-bottom: 4px;
+  }
+
+  &__code {
+    margin-bottom: 16px;
+
+    color: rgb(97, 96, 112);
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+  }
+
+  &__rate {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin-bottom: 24px;
+
+    &-text {
+      color: rgb(6, 11, 53);
+      font-size: 10px;
+      font-weight: 400;
+      line-height: 14px;
+    }
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    height: 18px;
+    margin-bottom: 16px;
+
+    &-text {
+      color: rgb(97, 96, 112);
+      font-size: 12px;
+      font-weight: 400;
+      line-height: 18px;
+      margin: 0;
+    }
+
+    &-money {
+      color: rgb(6, 11, 53);
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 18px;
+      margin: 0;
+    }
+  }
+
+  &__button {
+    color: rgb(255, 255, 255);
+    font-size: 14px;
+    font-weight: 700;
+
+    display: block;
+    text-decoration: none;
+    text-align: center;
+    line-height: 40px;
+
+    width: 100%;
+    height: 40px;
+
+    border-radius: 5px;
+    background: rgb(0, 172, 220);
+  }
+
+  &__button:hover {
+    color: rgb(0, 172, 220);
+
+    background: rgb(255, 255, 255);
+    font-size: 14px;
+    font-weight: 700;
+
+    display: block;
+    text-decoration: none;
+    text-align: center;
+    line-height: 40px;
+
+    width: 100%;
+    height: 40px;
+
+    box-sizing: border-box;
+    border: 1px solid rgb(0, 172, 220);
+    border-radius: 5px;
+  }
```

Comment:
it's redundant to repeat all styles in hover selector, write only those styles that you want to change on hover